docs/ru/query_language/system.md

```diff
@@ -0,0 +1,33 @@
+# SYSTEM Queries {#query_language-system}
```

Comment:
why not symlink?

Reply:
Because you will non be able to comment Russian translation pull request after I change the symlink with the file.